dist/index.js

```diff
@@ -4824,19 +4826,20 @@ async function getElixirVersion(exSpec0, otpVersion) {
   } else {
     throw new Error(
       `Requested Elixir / Erlang/OTP version (${exSpec0} / ${otpVersion}) not ` +
-        'found in version list',
+        "found in version list (should you be using option 'version-type': 'strict'?)",
     )
   }
 
-  return elixirVersionWithOTP
+  return `v${elixirVersionWithOTP}`
```

Comment:
And this is where we add it back. Mind you (https://repo.hex.pm/builds/elixir/builds.txt) all versions start with a `v` except for `master` (I'm not sure we want an exception for master, so keeping it out, for the time being).